README.md

```diff
@@ -0,0 +1,75 @@
+# HGCTPGEmulator
+
+## Overview
+
+Ensemble of python and C++ modules to run the HGC-TPG backend firmware emulator.
+
+## Dependencies
+
+The testing framework uses the `nlohmann_json` library, which must be installed and accessible. If not, it can be easily set, e.g. in a conda environment:
+
+```
+conda create --name newEnv
+conda activate newEnv
+conda install -c conda-forge nlohmann_json pkg-config
+```
+
+`pkg-config` is also needed inside the conda environment, in order to automatically pick up the package location inside the Makefile.
```

Comment:
Is it still needed since the json library is now included?

Reply:
updated README